src/managers/AdManager/AdManager.ts

```diff
@@ -21,82 +26,91 @@ class AdManager {
 
   private page: TPages | null;
   private productsIds: ReturnType<typeof getProductsIds> | [];
-  private websiteId: number;
 
   public injectAdnPixelScript = () => {
     const area = this.page ? this.mapPageToArea(this.page) : null;
 
     if (window.dlApi && dlApi.fetchNativeAd) {
-      dlApi.area = area || '';
-      dlApi.keyvalues = {
+      dlApi.addKeyValue('area', area || '');
+      dlApi.addKeyValue('keyvalues', {
```

Comment:
Należy zmienić parametry z którymi jest wywołana metoda `addKeyValue` aby zadziałała poprawnie na

```suggestion
      dlApi.addKeyValue('offer_ids', this.productsIds.toString())
```